Changes

```diff
@@ -3,6 +3,9 @@ Working version
 
 ### Language features:
 
+- #??: Let-punning
+  (Stephen Dolan, review by ??)
```

Comment:
I guess we need to refine this to "binding-operator punning". Could you include an actual example (and its desugaring)?

Reply:
Also, are you sure that it is let-punning not let-prunning?

Reply:
The name we use really is "punning". I think of it as a "pun" in that we use something for something else (the name of a declaration as its definition). The naming "record field punning" is well-established and more or less (well, remotely) consistent with [type punning](https://en.wikipedia.org/wiki/Type_punning) for example.

Reply:
haha, I was always under impression that I was using type *pruning*)) Good day I have learned something new)